src/test/kotlin/com/autonomousapps/internal/advice/AdvisorTest.kt

```diff
@@ -52,7 +54,8 @@ class AdvisorTest {
       unusedProcs = emptySet()
     )
     val computedAdvice = advisor.compute()
-    val changeAdvice = computedAdvice.advicePrinter().getChangeAdvice()
+    var consoleReport = build(computedAdvice)
```

Comment:
I think I like it better if you don't import the "static" function. Reading in github, it looks like a local function, and I prefer it "namespaced".